table/block_based/block_based_table_iterator.h

```diff
@@ -0,0 +1,833 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+#pragma once
+#include "table/block_based/block_based_table_reader.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+// Release the cached entry and decrement its ref count.
+inline void ForceReleaseCachedEntry(void* arg, void* h) {
```

Comment:
Minor: this could be moved to a `.cc` file.

Reply:
I'll move it.